[BUG] build error on tests

**Describe the bug**
I'm getting this build error when trying to build srt using the current git master:
```
[ 40%] Building CXX object CMakeFiles/srt-test-multiplex.dir/testing/testmedia.cpp.o
In file included from /build/srt-git/src/srt/testing/testmedia.cpp:26:
/build/srt-git/src/srt/srtcore/netinet_any.h: In constructor ‘sockaddr_any::sockaddr_any(int)’:
/build/srt-git/src/srt/srtcore/netinet_any.h:40:37: warning: ‘void* memset(void*, int, size_t)’ clearing an object of non-trivial type ‘struct sockaddr_any’; use assignment or value-initialization instead [-Wclass-memaccess]
   40 |         memset(this, 0, sizeof *this);
      |                                     ^
/build/srt-git/src/srt/srtcore/netinet_any.h:28:8: note: ‘struct sockaddr_any’ declared here
   28 | struct sockaddr_any
      |        ^~~~~~~~~~~~
/build/srt-git/src/srt/testing/testmedia.cpp: In constructor ‘UdpTarget::UdpTarget(std::string, int, const std::map<std::__cxx11::basic_string<char>, std::__cxx11::basic_string<char> >&)’:
/build/srt-git/src/srt/testing/testmedia.cpp:1275:43: error: conversion from ‘in_addr_t’ {aka ‘unsigned int’} to non-scalar type ‘in_addr’ requested
 1275 |             in_addr addr = maddr.sin_addr.s_addr;
      |                            ~~~~~~~~~~~~~~~^~~~~~
/build/srt-git/src/srt/testing/testmedia.cpp:1278:25: error: ‘status_error’ was not declared in this scope
 1278 |             if ( res == status_error )
      |                         ^~~~~~~~~~~~
make[2]: *** [CMakeFiles/srt-test-multiplex.dir/build.make:76: CMakeFiles/srt-test-multiplex.dir/testing/testmedia.cpp.o] Error 1
make[2]: Leaving directory '/build/srt-git/src/build'
make[1]: *** [CMakeFiles/Makefile2:102: CMakeFiles/srt-test-multiplex.dir/all] Error 2
make[1]: Leaving directory '/build/srt-git/src/build'
make: *** [Makefile:130: all] Error 2
make: Leaving directory '/build/srt-git/src/build'
```

**To Reproduce**
```
$ git clone https://github.com/Haivision/srt.git
$ cmake -B build -S srt -DENABLE_TESTING='True'
$ make -C build -j1
```

**Expected behavior**
Build success.

**Screenshots**
Not applicable.

**Desktop (please provide the following information):**
 - OS: Arch Linux x86_64
 - SRT Version / commit ID: git master, current at acd703f7348345248f7dfe68f1ead20c840c6250.
- Compiler: gcc 9.2.0

**Additional context**
- This is caused by commit acd703f7348345248f7dfe68f1ead20c840c6250.
- Reverting commit acd703f7348345248f7dfe68f1ead20c840c6250 "fixes" the issue.
- Disabling tests with `-DENABLE_TESTING='False'` also "fixes" the issue.

--- a/srtcore/common.cpp
+++ b/srtcore/common.cpp
@@ -410,14 +410,6 @@ m_iMinor(minor)
       m_iErrno = err;
 }
 
-CUDTException::CUDTException(const CUDTException& e):
-m_iMajor(e.m_iMajor),
-m_iMinor(e.m_iMinor),
-m_iErrno(e.m_iErrno),
-m_strMsg()
-{
-}
-
 CUDTException::~CUDTException()
 {
 }

--- a/srtcore/congctl.h
+++ b/srtcore/congctl.h
@@ -79,6 +79,7 @@ public:
     // 1. The congctl is individual, so don't copy it. Set NULL.
     // 2. The selected name is copied so that it's configured correctly.
     SrtCongestion(const SrtCongestion& source): congctl(), selector(source.selector) {}
+    void operator=(const SrtCongestion& source) { congctl = 0; selector = source.selector; }
 
     // This function will be called by the parent CUDT
     // in appropriate time. It should select appropriate

--- a/srtcore/udt.h
+++ b/srtcore/udt.h
@@ -247,7 +247,6 @@ class UDT_API CUDTException
 public:
 
    CUDTException(CodeMajor major = MJ_SUCCESS, CodeMinor minor = MN_NONE, int err = -1);
-   CUDTException(const CUDTException& e);
 
    ~CUDTException();
 

--- a/testing/testmedia.cpp
+++ b/testing/testmedia.cpp
@@ -1272,10 +1272,10 @@ public:
         if (adapter != "")
         {
             sockaddr_in maddr = CreateAddrInet(adapter, 0);
-            in_addr addr = maddr.sin_addr.s_addr;
+            in_addr addr = maddr.sin_addr;
 
             int res = setsockopt(m_sock, IPPROTO_IP, IP_MULTICAST_IF, reinterpret_cast<const char*>(&addr), sizeof(addr));
-            if ( res == status_error )
+            if (res == -1)
             {
                 Error(SysError(), "setsockopt/IP_MULTICAST_IF: " + adapter);
             }

